File: lib/src/main/java/com/hammwerk/wizardpager/MultiFixedChoicePage.java
package com.hammwerk.wizardpager;

import android.support.v4.app.Fragment;

import com.hammwerk.wizardpager.core.Page;

import java.util.List;

public class MultiFixedChoicePage extends Page<List<Integer>> {
	private String[] choices;

	public MultiFixedChoicePage(String title) {
		this(title, false);
	}

	public MultiFixedChoicePage(String title, boolean required) {
		super(title, required);
	}

	@Override
	protected Fragment createFragment() {
		return MultiFixedChoicePageFragment.createInstance(this, choices);
	}

	public MultiFixedChoicePage setChoices(String... choices) {
		this.choices = choices;
		return this;
	}
}

File: lib/src/main/kotlin/com/hammwerk/wizardpager/MultiFixedChoicePage.kt
package com.hammwerk.wizardpager

import com.hammwerk.wizardpager.core.Page

class MultiFixedChoicePage @JvmOverloads constructor(title: String, vararg var choices: String,
													 required: Boolean = false) :
		Page(title, MultiFixedChoicePageFragment::class.qualifiedName!!, required) {
	var selectedChoices: List<Int>? = null
}
